lib/taiko.js

```diff
@@ -381,6 +381,38 @@ module.exports.title = async () => {
     return result.result.value;
 };
 
+/**
+ * Clears browser cookies.
+ *
+ * @example
+ * clearBrowserCookies()
+ * 
+ * @returns {Promise<Object>} - Object with the description of the action performed.
+ */
+module.exports.clearBrowserCookies = async () => {
+    validate();
+    await network.clearBrowserCookies();
+    return { description: 'Browser cookies deleted successfully' };
+};
+
+/**
+ * Deletes browser cookies with matching name and url or domain/path pair.
+ *
+ * @example
+ * deleteCookies("CSRFToken", {url: "http://the-internet.herokuapp.com"})
+ *
+ * @param {string} cookieName - Cookie name.
+ * @param {Object} options
```

Comment:
A brief explanation of what options can be passed would be useful here. Also, an example for deleting cookies wrt to a domain can also be added.

Reply:
Done